packages/payment-integration-api/src/hosted-form/is-credit-card-form-fields.ts

```diff
@@ -0,0 +1,12 @@
+import HostedFieldType from './hosted-field-type';
+import { HostedCardFieldOptionsMap, HostedStoredCardFieldOptionsMap } from './hosted-form-options';
+
+export default function isCreditCardFormFields(
+    fields: HostedCardFieldOptionsMap | HostedStoredCardFieldOptionsMap,
+): fields is HostedCardFieldOptionsMap {
+    return (
+        HostedFieldType.CardNumber in fields &&
```

Comment:
Do we need to check for cardCode as well?

Reply:
Nope, because it is an optional param in TS interface